config.yaml

```diff
@@ -53,9 +53,9 @@
   
   # Sample specific info - REQUIRED
   fg_sample1: {
-                mate1: "mate1_name", # if file name is ENCFF041KJT.fastq.gz then add ENCFF041KJT 
```

Comment:
I guess the assumption is that the files are fastq.gz? This should be made clear. Any other format allowed here? It would actually be better to allow the user to plug in the input file name as is.